recipes/GPy/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "GPy" %}
+{% set version = "1.6.1" %}
+{% set sha256 = "7b8a87c660282eeba538b8c622b5f2b31e3b2cae8eb7c672c2a12b96923eeb1a" %}
+
+
+package:
+  name: "{{ name|lower }}"
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - numpy >=1.7
```

Comment:
Can you please add `toolchain` to the `build` requirements:

https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L33

Otherwise, everything else looks good to me.